SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -109,10 +109,13 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
     }
 
     @IBAction func placeCall(_ sender: UIButton) {
-        if (self.call != nil && self.call?.state == .connected) {
-            self.userInitiatedDisconnect = true
-            performEndCallAction(uuid: self.call!.uuid)
-            self.toggleUIState(isEnabled: false, showCallControl: false)
+        if (!self.activeCalls!.isEmpty) {
+            let call = self.activeCalls![0]
+            if (call.state == .connected) {
```

Comment:
You can simplify the code with ` if let call = self.activeCalls.first, call.state == .connected`